calendar-bundle/contao/templates/calendar/cal_mini.html5

```diff
@@ -2,13 +2,13 @@
 <table class="minicalendar">
 <thead>
   <tr>
-    <th class="head previous"><?php if ($this->prevHref): ?><a href="<?= $this->prevHref ?>" title="<?= $this->prevTitle ?>" data-skip-search-index><?= $this->prevLabel ?></a><?php else: ?>&nbsp;<?php endif; ?></th>
-    <th colspan="5" class="head current"><?= $this->current ?></th>
-    <th class="head next"><?php if ($this->nextHref): ?><a href="<?= $this->nextHref ?>" title="<?= $this->nextTitle ?>" data-skip-search-index><?= $this->nextLabel ?></a><?php else: ?>&nbsp;<?php endif; ?></th>
+    <th id="previous" class="head previous"><?php if ($this->prevHref): ?><a href="<?= $this->prevHref ?>" title="<?= $this->prevTitle ?>" data-skip-search-index><?= $this->prevLabel ?></a><?php else: ?>&nbsp;<?php endif; ?></th>
```

Comment:
This will create problems if there are multiple mini calendars on the same page. You should add the module ID as suffix.